Rovers/Rovers.tex

```diff
@@ -0,0 +1,405 @@
+part of space exploration. 
+This is also an area that RIT Space Exploration has very little experience with project-wise.
+The purpose of this project is to assess and assert the capability of RIT SPEX regarding the construction and fabrication of a mock-rover and then begin construction. 
+As this is a new area to RIT SPEX, prototyping and experimenting is necessary to most effectively accomplish this goal. 
+This project will look at the unknowns, technical challenges, project management, and member skills of RIT Space Exploration in regard to a University Rover Competition (URC) capable rover and rover construction. 
+The URC is hosted by The Mars Society annually. 
+It is very competitive and requires a very complex and expensive rover that is currently out of scope for SPEX. 
+This rover would need to be capable of full autonomous navigation, soil collection and analysis, and precise robotics. 
+This project is the first step in building the skills for such a team.
+Rovers are important, they are one of the primary ways we explore space and other planets. 
+This project exists to get our members experience with rovers by building such a rover. 
+
+\section{Primary Objective}
+\label{sec:primary-obj}
+The goal of this project is delivering a functioning mock rover.
+We define mock rover as a functioning rover built to specifications for a mock mission as it will not be launched. 
+We define rover as a vehicle for driving over rough terrain, especially one driven by remote control.
+The rover will allow the team to develop skills and knowledge to build a
+more feature completer rover in the future. 
+These skills include rover fabrication and design, autonomous navigation, electrical system design, rover testing, computer vision, and robotics.
+The rover will be testing using events designed like the URC but scaled down to fit the scope of this rover. 
+We will be using time as a metric as well as binary metrics for completion of the parts of the competition. 
+Some features might behave differently than expected or planned these features will be evaluated accordingly.
+This includes ’does the rocker-bogie system works as intended’, ’will the rover stop when directed at an obstacle’, etc. 
+The details are defined in the minimum viable product section.
+
+\subsection{Minimum Viable Product}
+\label{subsec:mvp}
+The minimum viable product (MVP) has been defined by the basics of what can be called a 'functioning rover'. This includes features like suspension, motors, power and controls. The MVP is defined by the following features: 
+
+\begin{table}[hb!]
+    \caption{Minimum Viable Product}
+    \centering
+    {\renewcommand{\arraystretch}{1.5}
+    \begin{tabularx}{\linewidth}{LL} 
+    \hline
+    \textbf{Feature} & \textbf{Reason} \\
+    \hline
+    Rocker-bogie-like suspension & This is the NASA preferred rover suspension style. \\
+    Ability to control remotely (such as a gamepad) & Remote control is essential for rovers. \\
```

Comment:
Why is it essential for rovers?

Could a pre-programmed rover complete the objectives? Will it be direct control over wheels or will software be written to 'fly by wire' to convert commands into intent?